Error with make serve

### What happened?

I followed the readme to start the local development server, but got an error.

```
marcus@NanElmoth pulumi-hugo % make serve-all
./node_modules/.bin/concurrently --kill-others -r "./scripts/serve.sh" "yarn --cwd ./themes/default/theme run start"
yarn run v1.22.19
$ yarn run concurrently 'yarn --cwd stencil run start' 'yarn run webpack --watch' --raw --kill-others
Start building sites … 
hugo v0.111.3+extended darwin/arm64 BuildDate=unknown
$ /Users/marcus/git/Comradin/pulumi-hugo/themes/default/theme/node_modules/.bin/concurrently 'yarn --cwd stencil run start' 'yarn run webpack --watch' --raw --kill-others
Error: Error building site: "/var/folders/sh/ldk5jf_95yb1yrrq79f17gn40000gn/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/pulumi/registry/themes/default@v0.0.0-20230316203047-8b0d4a0468c1/content/registry/packages/fusionauth/_index.md:16:1": failed to extract shortcode: shortcode "chooser" must be closed or self-closed
Built in 537 ms
make: *** [serve-all] Error 1
```

### Expected Behavior

The hugo server starts.

### Steps to reproduce

The things I did:

```
gh repo clone pulumi/pulumi-docs
cd pulumi-docs
make ensure
make serve
```

### Output of `pulumi about`

Not pulumi related but more a hugo or JavaScript module error.

### Additional context

Haven't tried other OSs, but this happend on macOS Ventura 13.2.1

```
% hugo version
hugo v0.111.3+extended darwin/arm64 BuildDate=unknown
%  go version
go version go1.20.2 darwin/arm64
% nvm version
v18.14.0
yarn install v1.22.19
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

--- a/themes/default/content/blog/kubernetes-architecture-templates/index.md
+++ b/themes/default/content/blog/kubernetes-architecture-templates/index.md
@@ -90,7 +90,7 @@ $ touch Pulumi.dev.yaml
 
 And do a quick cleanup of the `Pulumi.dev.yaml` file:
 
-{{< code-filename file="<root>/Pulumi.dev.yaml" >}}
+{{< code-filename file="<root>/Pulumi.dev.yaml" />}}
 
 ```yaml {.line-numbers}
 config:
@@ -106,7 +106,7 @@ Since we're running on GKE, we need to define a different namespace than `defaul
 
 We'll need to modify the `Pulumi.yaml` file to remove the virtual environment:
 
-{{< code-filename file="<root>/Pulumi.yaml" >}}
+{{< code-filename file="<root>/Pulumi.yaml" />}}
 
 ```yaml {.line-numbers}
 name: k8s-combo-infra
@@ -116,7 +116,7 @@ runtime: python
 
 And now we can set up our new `__main__.py` file! Create a new `__main__.py` file in the root of the repo, and add this code:
 
-{{< code-filename file="<root>/__main__.py" >}}
+{{< code-filename file="<root>/__main__.py" />}}
 
 ```python {.line-numbers}
 from pulumi import export as pulumi_export
@@ -141,7 +141,7 @@ We do need to make a couple small changes to the `app/__main__.py` file. We need
 1. pass that provider context to all of the resources, and
 1. get the endpoint from our app:
 
-{{< code-filename file="<root>/app/__main__.py" >}}
+{{< code-filename file="<root>/app/__main__.py" />}}
 
 ```python {.line-numbers hl_lines=[5,"14-19",27,51,93,109,111,117]}
 import pulumi
@@ -273,7 +273,7 @@ $ poetry run pulumi up
 
 And up they all go! Note that this takes a while, so don't panic!
 
-{{< code-filename file="output" >}}
+{{< code-filename file="output" />}}
 
 ```bash
 Previewing update (dev)

--- a/themes/default/content/blog/pulumi-azure-2-0/index.md
+++ b/themes/default/content/blog/pulumi-azure-2-0/index.md
@@ -178,6 +178,7 @@ pulumi-azure>=2.0.0
 ```
 
 {{% /choosable %}}
+{{% /chooser %}}
 
 There are quite a few breaking changes, so you may need to adjust your code before you can run the program successfully again.
 

